Tools/Scripts/build-webkit

```diff
@@ -235,6 +235,13 @@ if (isAppleCocoaWebKit()) {
         # Ignore any workspace set by set-webkit-configuration
         overrideConfiguredXcodeWorkspace("");
     }
+
+    if (determineXcodeVersion() lt "15.0" && !defined($ENV{UseSRCROOTSupportForTAPI})) {
+        # The call to XcodeOptions below checks this environment variable and
+        # sets TAPI_USE_SRCROOT for xcconfigs.
+        $ENV{UseSRCROOTSupportForTAPI} = "YES";
```

Comment:
Confused about why this is needed in `build-webkit`, I expected `webkitdirs.pm` to determine the options in this case. But also, I expected xcconfigs to seethes when appropriate.

Reply:
In shipping Xcode, the only way to turn on SRCROOT support is via this environment variable or a defaults key. The defaults key is in the `com.apple.dt.XCBuild` domain, _not_ com.apple.dt.Xcode, meaning we can't pass it as an xcodebuild argument. So for testing in EWS, we either need to set it here, or in the buildbot command line.

I can come up with a way to do this in our Makefiles, too, but it'll need to be handled separately.